Test/typewise_alert.test.py

```diff
@@ -0,0 +1,26 @@
+import unittest
+import sys
+import os, json
+import subprocess
+
+sys.path.append(os.path.join(os.path.dirname(__file__), os.pardir))
+import src.typewise_alert as typewise_alert 
+from src.typewise_alert import TypeWise_Alert as TWA
+
+
+class TypewiseTest(unittest.TestCase):
+
+  def test_check_and_alert(self):
+    root_dir = TWA().get_root_dir()
+    with open(os.path.join(root_dir, "inc", "test_case.json"), "r") as jsonread:
+      json_data = json.load(jsonread)
+    
+    for dt in json_data["check_and_alert"]:
```

Comment:
Good approach for configuring test values 👍 
But how do you verify if controller or email target has received these messages?

Reply:
Hi @prithamvshetty I thought we needed to just print the output in the console
do we need to add external libraries for it?

Reply:
Yes, the requirement could be anything such as printing to console or to a file. 
Verifying functionality is one aspect and getting coverage is one more. If we have good verification tests, then that automatically covers coverage part...
Hence I asked about verification.
By the way, we know from code review that the test will provide coverage 😊

Reply:
ohk thankyou